cmd/site-controller/README.md

```diff
@@ -0,0 +1,54 @@
+# Skupper Site Controller
+
+The site controller provides declarative methods to manage a Skupper VAN site using:
+
+* Kubernetes ConfigMaps
+* Tokens
+
+
+## Managing a Skupper Site using ConfigMaps
+
+ConfigMaps allow you to manage a Skupper site using the following parameters:
+
+`metadata:name` -  the site controller name in OpenShift
+
+`data:name` -  the VAN deployment or site name
+
+`data:cluster-local` -  (true/false) Set up skupper to only accept connections from within the local cluster.
+
+`data:console` -  (**true**/false) Enable skupper console
+
+`data:console-authentication` -  ('openshift', 'internal', 'unsecured') Autentication method
+
+`data:console-user` -  username for 'unsecured' option
+
+`data:console-password` - password for 'unsecured' option
+
+`data:edge` -  (true/false) Set up an edge skupper site
+
+`data:router-console` - (true/false) Set up a Dispatch Router console (not recommended)
+
+`data:service-controller` - 
```

Comment:
@ajssmith is this an optional service controller equivalent to the 'proxy-controller'?

What effect does this have for users?

Reply:
Yes, it is equivalent to --enable-proxy-controller in the cli. 

It would be for a scenario where you want to deploy the router but do not want/need the service controller to be running.